package.json

```diff
@@ -15,11 +15,12 @@
   },
   "main": "index.js",
   "dependencies": {
+    "@wmde/wikit-tokens": "1.0.1",
+    "@wmde/wikit-vue-components": "1.0.1",
     "core-js": "^3.6.5",
+    "npm": "^6.14.8",
```

Comment:
This line shouldn't be there. The version is fixed in the Dockerfile and npm is not a run-time dependency either.

Reply:
Fixed now.